RSA_Algorithm/README.md

```diff
@@ -0,0 +1,93 @@
+# **RSA Algorithm**
+### RSA (Rivest-Shamir-Adleman) is an algorithm used by modern computers to encrypt and decrypt messages. It is an asymmetric cryptographic algorithm. Asymmetric means that there are two different keys. This is also called public key cryptoraphy, because one of the keys can be given to anyone. The other key must be kept private. The algorithm is based on the fact that finding the factors of large composite number is difficult: when the factors are prime numbers, the problem is called prime factorization. It is also a key pair (public and private key) generator.
+
+![rsa algorithm](https://www.educative.io/api/edpresso/shot/5284561120395264/image/5318859621924864)
+
+## **Algorithm**
+---
+- ### **Generating the keys**
+  ### 1.Generate two large random primes, p and q, of approximately equal size such that their product n=pq is of the required bit length.
+  ### 2.Compute n=pq and ϕ=(p−1)(q−1).
+  ### 3.Choose an integer e, 1<e<ϕ, such that gcd(e,ϕ)=1.
+  ### 4.Compute the secret exponent d, 1<d<ϕ, such that ed≡1modϕ. 
+  ### 5.The public key is (n,e) and the private key (d,p,q). Keep all the values d, p, q and ϕ secret. [Sometimes the private key is written as (n,d) because you need the value of n when using d. Other times we might write the key pair as ((N,e),d).]
+
+## **Encryption**
+---
+### Given a plaintext P, represented as a number, the ciphertext C is calculated as:
+### C = P^{e} mod n
```

Comment:
please remove the '#' sign.

Reply:
@asha15  Required changes have been done.